packages/trip-details/src/index.tsx

```diff
@@ -181,7 +181,10 @@ export function TripDetails({
     fare = (
       <S.Fare>
         <TransitFareWrapper>
-          <summary style={{ display: fareKeys.length > 1 ? "list-item" : "" }}>
+          <summary
+            role="button"
```

Comment:
Add an `aria-expanded` attribute. 

I would disagree with `role="button"` for the `<summary>` element, so maybe we shouldn't use `<summary>`? Also, in another PR, we'll need to implement `aria-controls` or `aria-owns` for this expandable component.

Reply:
I followed https://codepen.io/scottohara/pen/aaJXYG. I think this is very much a summary but also think that there may be a better way than a random codepen to make it properly accessible

Reply:
The component that reacts to clicking should be the button element... but I'm not sure which one it is!

Reply:
And we'll have to fix this. Basically, the `<summary>` should be outside of `<details>`.
<img width="220" alt="image" src="https://user-images.githubusercontent.com/56846598/163588561-0bb4539d-8d27-4c30-8b79-229603f2e75e.png">